lib/talkable/generators/templates/app/views/shared/_talkable_offer.html.erb

```diff
@@ -1,3 +1,3 @@
-<%- if @offer %>
-  <%== @offer.advocate_share_iframe %>
+<%- if offer %>
+  <%== offer.advocate_share_iframe(defined?(options) ? options : {}) %>
```

Comment:
The best way to handle optional partial argumet is `options || = {}` at the beginning of the file.